manifests/cinder-csi-plugin/cinder-csi-controllerplugin.yaml

```diff
@@ -25,7 +25,7 @@ spec:
       serviceAccount: csi-cinder-controller-sa
       containers:
         - name: csi-attacher
-          image: k8s.gcr.io/sig-storage/csi-attacher:v3.4.0
+          image: k8s.gcr.io/sig-storage/csi-attacher:v4.0.0
```

Comment:
we have to add --default-fstype=ext4 explicitly here ?
https://github.com/kubernetes-csi/external-attacher/blob/release-4.0/CHANGELOG/CHANGELOG-4.0.md#no-really-you-must-read-this-before-you-upgrade

Reply:
@ramineni good point, will update